applications/ray/kuberay-tpu-webhook/VERSION

```diff
@@ -0,0 +1,4 @@
+# Release version of the Ray TPU webhook
+# Increment this value as part of any change to the webhook.
+# This value is separate from the ai-on-gke release version.
+v1.1.0
```

Comment:
I thought we were using tags instead? Keeping track of a version in a file will be difficult to track and easy to override existing versions

Reply:
That makes sense, I was just having trouble finding a way to create a git tag that would apply to just the webhook. I talked to Spencer and he said it was alright to create `kuberay-tpu-webhook/vX.Y.Z` tags that apply to the whole repo, so I'll change it to use that instead of the VERSION file.